.dependabot/config.yml

```diff
@@ -4,6 +4,31 @@ update_configs:
   - package_manager: 'javascript'
     directory: '/'
     update_schedule: 'weekly'
+    automerged_updates:
+      - match:
+          dependency_name: 'chai*'
+          update_type: 'semver:minor'
+      - match:
+          dependency_name: 'cypress'
+          update_type: 'semver:minor'
+      - match:
+          dependency_name: 'eslint*'
+          update_type: 'semver:minor'
+      - match:
+          dependency_name: 'enzyme*'
+          update_type: 'semver:minor'
+      - match:
+          dependency_name: 'mocha*'
+          update_type: 'semver:minor'
+      - match:
+          dependency_name: 'sazerac'
+          update_type: 'semver:minor'
+      - match:
+          dependency_name: 'sinon*'
+          update_type: 'semver:minor'
+      - match:
+          dependency_name: 'snap-shot-it'
+          update_type: 'semver:minor'
```

Comment:
In general, I think this is a fairly sensible set of patterns to auto-merge if the tests pass, but obviously I'm open to feedback on it and it can evolve as time goes on..

The only thing I'm really checking when I review PRs for these packages is whether the build is green, so I figure we might as well automate that rule instead of having to do it myself. Major version updates will still require manual review. Even if a bad update does slip the net, only testing/linting tools will break - no production dependencies.

Reply:
This seems like a reasonable set to me as well. I still do a quick manual spot check on the review apps even for these dev-only deps, but acknowledge that the E2E test job really covers that scenario anyway.